### PR TITLE
600W silicone heater thermistor

### PR DESCRIPTION
**Pull request by salfter**
### Description

(Revised...patch is against bugfix-2.0.x, not 2.0.x.)

A while back, I purchased this 600W silicone bed heater:

https://www.aliexpress.com/item/32859679498.html

The thermistor within doesn't seem to match any that Marlin knows about; no matter what I selected, it tended to read low. (I tried bringing it up to 85 one time, and it started to smoke a little bit!) I knocked together a rig to correlate the ADC readings for this thermistor with those from a known thermistor, and put the results in a new thermistor table.

### Benefits

It keeps my bed heater temperature where it should be, at least within the usual range (60-110) within which I'll run it.

### Related Issues

None that I know about.

### CODE REVIEW

File: Marlin/src/module/thermistor/thermistor_76.h [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  { OV( 926),  60 },
+  { OV( 938),  55 },
+  { OV( 947),  50 },
+  { OV( 954),  45 },
+  { OV( 961),  40 },
+  { OV( 967),  35 },
+  { OV( 974),  30 },
+  { OV( 979),  25 },
+  { OV(1000),  10 },
+  { OV(1010),   0 }
```
Comment (thinkyhead) [inline Marlin/src/module/thermistor/thermistor_76.h:64]: You may be able to reduce the temperature with a tray of ice to observe this region of the sensor.

**Comment by sjasonsmith** [comment]
Is adding tables for mystery thermistors a good idea? If it is unlikely to be very reusable by other people, maybe it is best kept as a custom table for an individual user?

**Comment by thinkyhead** [comment]
My hope is that the actual model number may be identified, if it is indeed unique.

**Comment by thinkyhead** [comment]
Seems similar but clearly not the same as 202, which is also "unknown" but a standard part in a CopyMaster printer.

I agree that a thermistor which is not a standard part of a common heater pad product should not be added. Manufacturers should be supplying a proper thermistor table if their product is meant to be used in a Marlin-based 3D printer.

That said, it should be noted that there is no extra cost in compile-time for adding a new Thermistor table header.

**Comment by salfter** [comment]
> maybe it is best kept as a custom table for an individual user?

I wouldn't mind this approach, but the manufacturer didn't provide the thermistor beta, and there seems to be no provision for a table, just resistance at 25°C and beta.

With the calculator at [this page](https://www.ametherm.com/inrush-current/calculators) and a few cycles of testing, it looks like a beta value of 2681 is at least in the ballpark (within 1°C).  A quick scan of the Marlin source returns beta values from 3950 to 4700, so this definitely doesn't match up with any of those.

**Pull request closed without merging**